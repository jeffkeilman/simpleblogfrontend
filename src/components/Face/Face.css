.Face {
  display: inline-block;
  width: 30%;
  height: auto;
}

.Face img {
  width: 100%;
}