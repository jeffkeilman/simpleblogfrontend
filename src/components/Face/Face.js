import './Face.css'
import faceImage from './assets/Face.jpg'

function Face () {
  return (
    <section className='Face'>
      <img
        src={faceImage}
        alt="Author's face"
      />
    </section>
  )
};

export default Face
