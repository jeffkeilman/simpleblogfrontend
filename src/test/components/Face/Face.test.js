import React from 'react'
import renderer from 'react-test-renderer'
import Face from '../../../components/Face/Face'
import { render } from '@testing-library/react'

describe('Face', () => {
  // snapshot test
  it('renders correctly', () => {
    const face = renderer.create(<Face />).toJSON()
    expect(face).toMatchSnapshot()
  })

  it('renders face image', () => {
    const { queryByAltText } = render(<Face />)
    expect(queryByAltText(/Author's face/i)).toBeInTheDocument()
  })
})
