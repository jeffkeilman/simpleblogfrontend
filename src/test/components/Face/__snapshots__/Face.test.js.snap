// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Face renders correctly 1`] = `
<section
  className="Face"
>
  <img
    alt="Author's face"
    src="Face.jpg"
  />
</section>
`;
